src/igvfd/schemas/user.json

```diff
@@ -86,12 +86,30 @@
         "job_title": {
             "title": "Job title",
             "type": "string",
-            "pattern": "^(\\S+(\\s|\\S)*\\S+|\\S)$"
+            "uniqueItems": true,
+            "permission": "import_items",
+            "items": {
+                "type": "string",
+                "enum": [
+                    "Primary Investigator",
+                    "Project Manager",
+                    "Submitter",
+                    "Post Doc",
+                    "Data Wrangler",
+                    "Scientist",
+                    "Computational Scientist",
+                    "Software Developer",
+                    "Other"
```

Comment:
@khzlin do we have a lot of graduate students or students as users?  I am also wondering what we call the NHGRI folks.  What do we call them now?

Reply:
I think "trainees" are used quite a lot to describe students a lot in calls. As for NHGRI folks, I'm not sure how to call them. Maybe "NHGRI Scientific Program Analyst"?

Reply:
I would like to NOT use Trainees - due to a discussion of how to call the "young generation of researchers" that is happening at slack for IGVF

Reply:
What about NHGRI Staff Member and is Student offensive?  I think we can add Post Doc.

Reply:
I like "NHGRI Staff Member". Is it OK to proceed without a term for Student for now? @gabdank @cricketsloan

Reply:
yes we can leave out student and get this done